qdrant-landing/content/documentation/faq/qdrant-fundamentals.md

```diff
@@ -36,11 +36,8 @@ What Qdrant can do:
 - Apply full-text filters to the vector search (i.e., perform vector search among the records with specific words or phrases)
 - Do prefix search and semantic [search-as-you-type](../../../articles/search-as-you-type/)
 - Sparse vectors, as used in [SPLADE](https://github.com/naver/splade) or similar models
-
-What Qdrant plans to introduce in the future:
-
-- ColBERT and other late-interaction models
-- Fusion of the multiple searches
+- [Multi-vectors](../../concepts/vectors/#multivectors) for ColBERT and other late-interaction models
+- Fusion of the [multiple searches](../../concepts/hybrid-queries/)
```

Comment:
```suggestion
- Combination of the [multiple searches](../../concepts/hybrid-queries/)
```

Fusion suggests using something like RRF (reranking based on the scores returned by the individual methods), however we can also implement some other techniques.